docs/data/date-pickers/custom-components/custom-components.md

```diff
@@ -231,6 +231,18 @@ You can pass custom components—to replace the icons, for example—as shown be
 
 {{"demo": "ArrowSwitcherComponent.js", "defaultCodeOpen": false}}
 
+## Popper
+
+You can customize the popper that wraps the picker views the same way you would customize the [Material UI Popper](/material-ui/react-popper/)
+
+:::info
+When the picker views have different heights, there might be a layout shift if there is not enought space in the viewport for one of the views **below** the input field.
```

Comment:
I would also be in favor of back-linking the issues that this can solve so that people could easily reference, what would this solution actually solve.
What do you think?